ods_ci/tests/Resources/Files/workbench-runtimes/update-runtimes.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#######################################################################################
+# Follow these steps to update the workbench runtimes in this folder                  #
+#  - Open a browser and connect to RHOAI Dashboard                                    #
+#  - Create a data science project my-project                                         #
+#  - Start a Workbench using the Standard Data Science Image. Name should be wb-sds   #
```

Comment:
Why the dashboard team can't have a tag that matches the RHOAI releases?

Reply:
@jstourac what are your thoughts about ^ ?

Reply:
> Why the dashboard team can't have a tag that matches the RHOAI releases?
I presume it should be IDE team, not dashboard team.

I guess that the reason we don't use RHOAI version tags for the images is that we don't necessarily update our images with every single rhoai release - as such, these aren't exactly bounded to one and only release now but are being updated based on the changes being done in the repository.

Reply:
Thanks for the explanation. It makes sense